controllers/suite_test.go

```diff
@@ -468,7 +471,7 @@ func installYamlResource(resPath string) error {
 	cobj := robj.(client.Object)
 	cobj.SetNamespace(defaultNamespace)
 
-	if oprImg := os.Getenv("OPERATOR_IMAGE"); oprImg != "" {
+	if oprImg := os.Getenv("IMG"); oprImg != "" {
```

Comment:
Should `IMG` be `OPERATOR_IMAGE_REPO`?
```suggestion
	if oprImg := os.Getenv("OPERATOR_IMAGE_REPO"); oprImg != "" {
```

Reply:
We should use IMG as in the Makefile we define the IMG as
IMG ?= $(OPERATOR_IMAGE_REPO):$(OPERATOR_VERSION)
So we running the test you need to define OPERATOR_IMAGE_REPO and OPERATOR_VERSION